cardano-db-sync/src/Cardano/DbSync/Era/Universal/Insert/GovAction.hs

```diff
@@ -124,17 +128,39 @@ insertGovActionProposal cache blkId txId govExpiresAt (index, pp) = do
           , DB.treasuryWithdrawalAmount = Generic.coinToDbLovelace coin
           }
 
-    insertNewCommittee gaId removed added q = do
-      void . DB.insertNewCommittee $
-        DB.NewCommittee
-          { DB.newCommitteeGovActionProposalId = gaId
-          , DB.newCommitteeQuorumNumerator = fromIntegral $ numerator r
-          , DB.newCommitteeQuorumDenominator = fromIntegral $ denominator r
-          , DB.newCommitteeDeletedMembers = textShow removed
-          , DB.newCommitteeAddedMembers = textShow added
+    insertNewCommittee ::
+      DB.GovActionProposalId ->
+      Set (Ledger.Credential 'ColdCommitteeRole StandardCrypto) ->
+      Map (Ledger.Credential 'ColdCommitteeRole StandardCrypto) EpochNo ->
+      UnitInterval ->
+      ReaderT SqlBackend m ()
+    insertNewCommittee gapId removed added q = do
+      insertNewCommitteeInfo gapId q
+      insertMembers gapId removed added q
+
+    insertNewCommitteeInfo gapId q =
+      void . DB.insertNewCommitteeInfo $
+        DB.NewCommitteeInfo
+          { DB.newCommitteeInfoGovActionProposalId = gapId
+          , DB.newCommitteeInfoQuorumNumerator = fromIntegral $ numerator r
+          , DB.newCommitteeInfoQuorumDenominator = fromIntegral $ denominator r
           }
       where
         r = unboundRational q -- TODO work directly with Ratio Word64. This is not currently supported in ledger
+    insertMembers gapId removed added q = do
```

Comment:
not sure if this is a diff view error but doesn't look like these following 2 functions are indented correctly?

Reply:
It looks good to me. `insertNewCommittee`, `insertNewCommitteeInfo` and `insertMembers` are defined in the same level. The first calls the other 2.